lua/starfall/libs_sh/game.lua

```diff
@@ -1,7 +1,27 @@
 -- Global to all starfalls
+local checkluatype = SF.CheckLuaType
 local registerprivilege = SF.Permissions.registerPrivilege
 
-if SERVER then registerprivilege("blast.create", "Blast damage", "Allows the user to create explosions", { usergroups = { default = 1 } }) end
+if SERVER then
+	registerprivilege("blast.create", "Blast damage", "Allows the user to create explosions", { usergroups = { default = 1 } })
+	registerprivilege("bullets.fire", "FireBullets", "Allows the user to fire bullets", {})
+end
+
+local fireBulletsBurst = SERVER and SF.BurstObject("fireBullets", "firebullets", 40, 40, " bullets can be fired per second", "Number of bullets that can be fired in a short time")
+local fireBulletsDPSBurst = SERVER and SF.BurstObject("fireBulletsDPS", "bullets dps", 100, 100, " maximum bullets damage per second", "Damage per second bullets can deal")
+
+local maxBulletDamage = SERVER and CreateConVar("sf_bullets_maxdamage", 100, FCVAR_ARCHIVE, "Maximum amount of damage a bullet can deal", 1)
+local maxBulletForce = SERVER and CreateConVar("sf_bullets_maxforce", 100, FCVAR_ARCHIVE, "Maximum amount of force a bullet can have", 0)
+local maxBulletHull = SERVER and CreateConVar("sf_bullets_maxhull", 10, FCVAR_ARCHIVE, "Maximum hull size a bullet can have", 0)
+local maxBulletNum = SERVER and CreateConVar("sf_bullets_maxnum", 5, FCVAR_ARCHIVE, "Maximum amount of bullets that can be fired at the same time", 1)
+
+local allowedAmmoType = SERVER and {
```

Comment:
Maybe just reference the enum instead?

Reply:
Users can modify the enum table so that wouldn't work, but I will see if this option should even exist.